src/axom/quest/IntersectionShaper.hpp

```diff
@@ -499,6 +504,10 @@ class IntersectionShaper : public Shaper
     m_hexes = axom::allocate<PolyhedronType>(NE);
     m_hex_bbs = axom::allocate<BoundingBoxType>(NE);
 
+    // Oddities required by hip
```

Comment:
See my comment at (new) line 332.  Possibly use the following, if you want, and if it's accurate:
```suggestion
    // Oddities required by hip to avoid capturing `this`
```